marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid

```diff
@@ -0,0 +1,181 @@
+
+#### init
+Initialize required directory structure.
+
+In order to correctly communicate with the platformOS engine and API, your codebase should be organized into a specific directory structure. Use the `marketplace-kit init` command to download the directory structure in a zip file, and extract it into the current directory. 
+
+| Command 	| Usage                 	| Example               	|
+|---------	|-----------------------	|-----------------------	|
+| init   	| `marketplace-kit init` 	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit init</code></pre> 	|
+
+**Options**:
+
+* `--url <url>`: theme GitHub repository URL (default: "https://github.com/mdyd-dev/directory-structure")
+* `--branch <branch>`: branch where the theme is located (default: "master")
+
+**Related topics**: 
+
+* [Creating the Codebase](/get-started/creating-codebase)
+
+#### logs
+Attach to environment log streams. 
+
+| Command 	| Usage                 	| Example               	|
+|---------	|-----------------------	|-----------------------	|
+| logs   	| `marketplace-kit logs` 	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit logs</code></pre> 	|
+
+**Options**: 
+
+*  `-c --config-file <config-file>`: config file path (default: ".marketplace-kit")
+* `--interval <interval>`: time to wait between updates in ms (default: 3000)
+
+#### migrations
+Generate or run a migration. 
+
+**Operations**: 
+
+* `generate <environment> <name>`: generate new empty migration
+* `run <environment> <name>`: run migration on environment
+* `list <environment> <name>`: list migrations and their statuses 
+
+| Command      	| Usage                                     	| Example                                 	|
+|--------------	|-------------------------------------------	|-----------------------------------------	|
+| migrations generate  	| `marketplace-kit migrations generate <environment> <name>` 	|  <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit migrations generate local add_some_books</code></pre> 	|
+| migrations run  	| `marketplace-kit migrations run <environment> <name>` 	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit migrations run local 20180909123423</code></pre>  	|
+| migrations list   	| `marketplace-kit migrations list <environment> <name>`  	|       	|
+
+**Related topics**: 
+
+* [Migrating Data](/tutorials/migrations/migrating-data)
+
+#### sync
+Update environment on file change. 
+
+Enables sync mode: immediately pushes changes made to the codebase to the proper environment. It feels like working on localhost. It is recommended to use sync only for staging environments, as pushing changes immediately to production can be dangerous.  
```

Comment:
Well, its not pushing changes - it will check whats different between server and local and sync it - it will only send files after local change and then send only those files that changed after sync has been started.

Reply:
Thank you for the clarification, I edited it.